tools/pylib/boutdata/mms.py

```diff
@@ -321,7 +321,7 @@ def write(self, nx, ny, output):
 
         """
 
-        MXG = 2
+        MXG = MXG
```

Comment:
yes of course. Sorry. I am not that familiar with the fork-branch-pull_request workflow.... Am I supposed to make make the changes and make a new pull request ?